### ISSUE TITLE
Panic on `devspace remove deployment --all`

### ISSUE THREAD

**Issue by FabianKramm** [devspace-cloud/devspace#645]
A panic is thrown on `devspace remove deployment --all`

```
$ devspace remove deployment --all
[info]   Loaded config devspace.yaml
? Do you want to delete all deployment resources deployed?
 no
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/devspace-cloud/devspace/cmd/remove.(*deploymentCmd).RunRemoveDeployment(0xc0004530a9, 0xc000dbd400, 0xc0010a1e80, 0x0, 0x1)
        /Users/fabiankramm/Programmieren/go-workspace/src/github.com/devspace-cloud/devspace/cmd/remove/deployment.go:105 +0x912
github.com/spf13/cobra.(*Command).execute(0xc000dbd400, 0xc0010a1e60, 0x1, 0x1, 0xc000dbd400, 0xc0010a1e60)
        /Users/fabiankramm/Programmieren/go-workspace/src/github.com/devspace-cloud/devspace/vendor/github.com/spf13/cobra/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x76b24c0, 0x5c46840, 0xc0003e2a00, 0xc000519640)
        /Users/fabiankramm/Programmieren/go-workspace/src/github.com/devspace-cloud/devspace/vendor/github.com/spf13/cobra/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/fabiankramm/Programmieren/go-workspace/src/github.com/devspace-cloud/devspace/vendor/github.com/spf13/cobra/command.go:800
github.com/devspace-cloud/devspace/cmd.Execute()
        /Users/fabiankramm/Programmieren/go-workspace/src/github.com/devspace-cloud/devspace/cmd/root.go:55 +0xe8
```

**Comment by 1522784** [comment]
The same goes for "devspace remove image --all". Here it is even worse because the Done-message doesn't tell you whether the deleted image was there before the deletion.

**Comment by FabianKramm** [comment]
Fixed in v3.5.18
